backend/app/api/v2/commons/common.py

```diff
@@ -25,6 +25,7 @@ async def getData(start_datetime: date, end_datetime: date):
     await es.close()
     tasks = [item['_source'] for item in response["hits"]["hits"]]
     jobs = pd.json_normalize(tasks)
+    jobs[['nodeName', 'benchmark']] = jobs[['nodeName', 'benchmark']].fillna('')
```

Comment:
Can you make it generic?
I mean all NaN values would be replaced with NaN. Is that possible?